### DISCUSSION TITLE
Node version changes

### DISCUSSION THREAD

**Discussion by lunchboxer** [beginner-corp/begin-community#116] [Support] [answered]
When and how does the version of node for Begin apps change? When I run process.version I see that my app is currently run on v14.20. When I look at AWS Lamda runtimes, I notice that node 18 and 16 are also supported. Is is possible to set up new Begin apps with them?

**Comment by ryanblock** [comment, marked as answer]
Hey there! Just a heads up: we've put the legacy Begin in maintenance/security patch only mode for now (see: https://blog.begin.com/posts/2022-08-31-new-begin-and-enhance-html-framework). Thanks for the reminder on `nodejs16.x`, will go upgrade that today. Unfortunately, because `nodejs18.x` introduces breaking changes, we likely won't be supporting that in legacy Begin apps, but we do support it out of the box in the version of Begin that will be shipping to customers in the coming months.

**Comment by ryanblock** [comment]
Update: patched legacy Begin, fresh app deploys will now run `nodejs16.x`. Thanks!
